src/ConfigProvider.php

```diff
@@ -14,13 +14,15 @@
  * Configuration provider for the package.
  *
  * @see https://docs.laminas.dev/laminas-component-installer/
+ *
+ * phpcs:disable WebimpressCodingStandard.PHP.DisallowFqn.FileName
```

Comment:
Reported in webimpress/coding-standard#118

`JSON_PRETTY_PRINT` is not detected as a used contants.

Reply:
@geerteltink Not sure how `WebimpressCodingStandard.PHP.DisallowFqn.FileName` is related to the issue you have reported ;-)

Reply:
@michalbundyra Ah, I tagged the wrong line. This is causing:

```
FILE: src/ConfigProvider.php
----------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 3 ERRORS AFFECTING 3 LINES
----------------------------------------------------------------------------------------------------------------------------------------------------------
 38 | ERROR | Zend\ProblemDetails\ProblemDetailsMiddleware must be imported but file with name ProblemDetailsMiddleware exists in the namespace
 39 | ERROR | Zend\ProblemDetails\ProblemDetailsNotFoundHandler must be imported but file with name ProblemDetailsNotFoundHandler exists in the namespace
 40 | ERROR | Zend\ProblemDetails\ProblemDetailsResponseFactory must be imported but file with name ProblemDetailsResponseFactory exists in the namespace
----------------------------------------------------------------------------------------------------------------------------------------------------------
```

Reply:
What I've done in other packages is to import these classes at the top of the file, but with the prefix "Legacy":

```php
use Zend\ProblemDetails\ProblemDetailsResponseFactory as LegacyProblemDetailsResponseFactory;
```

When you reference them later:

```php
LegacyProblemDetailsResponseFactory::class
```

will then work, and you no longer get those phpcs issues. It also makes it clear _why_ we are importing them.